src/Elastic.Markdown/IO/DocumentationSet.cs

```diff
@@ -51,6 +51,10 @@ public DocumentationSet(BuildContext context, ILoggerFactory logger, ICrossLinkR
 		Files = [.. context.ReadFileSystem.Directory
 			.EnumerateFiles(SourcePath.FullName, "*.*", SearchOption.AllDirectories)
 			.Select(f => context.ReadFileSystem.FileInfo.New(f))
+			.Where(f => !f.Attributes.HasFlag(FileAttributes.Hidden) && !f.Attributes.HasFlag(FileAttributes.System))
+			.Where(f => !f.Directory!.Attributes.HasFlag(FileAttributes.Hidden) && !f.Directory!.Attributes.HasFlag(FileAttributes.System))
+			// skip hidden folders
+			.Where(f => !Path.GetRelativePath(SourcePath.FullName, f.FullName).StartsWith('.'))
```

Comment:
@reakaleek this is actually an important fix for docs-content and asciidocalypse since they generate from the root 😸